bubblesub/data/hotkeys.conf

```diff
@@ -59,6 +59,7 @@ Alt+J              sub-merge --concat
 Alt+Up             sub-move --above
 Alt+Down           sub-move --below
 Alt+Return         file-properties
+F5                 reload-cmds
```

Comment:
That choice is pretty intrusive IMO, and since it's a subtitle editor I'd expect "refresh" operation to reload the subtitles from disk (which we do not support), not reload the program plugins.

I'd choose something with Ctrl+ such as Ctrl+F10 or something, that's obviously "custom". Maybe other software has this shortcut?

Reply:
Nope, I've chosen F5 because it's usually the most common shortcut to reload something. What about Ctrl+F5?

Reply:
> That choice is pretty intrusive IMO, and since it's a subtitle editor I'd expect "refresh" operation to reload the subtitles from disk (which we do not support), not reload the program plugins.

Ok, I agree with using the F5 shortuct as a refresh option for subtitles. (Do you want to implement that feature by any chance? Because it's pretty useless imho.)

Reply:
>Do you want to implement such feature

Yes – sometimes I do some batch edits in Vim for the files I happen to have currently loaded in bubblesub, and I need to locate them again in the file open dialog

Reply:
Ok, great! So I'm going to change F5 to Ctrl+F5 :)

Reply:
I still don't like this choice. Ctrl+F5 is what I would envision for the subtitle reload.
I'm against single-button global hotkeys in general.

Reply:
Let's use Ctrl+F10 then. Anyway, there is another single-button global hotkey

`F3                 search-repeat --below`

What do you want to do with this one? Ctrl+F3 for below and Shift+F3 for above?

Reply:
This is fine because it doesn't do anything intrusive such as rescanning the whole plugin directory, unloading stuff, loading it again, possibly spamming the console with errors if there is syntax error in the changed files, and is generally a feature that is unused by the majority of the users yet it occupies such a popular shortcut